pkg/metrics/metricsconfig/initmetrics.go

```diff
@@ -49,6 +50,7 @@ func InitAllMetrics(registry *prometheus.Registry) {
 		eventcache.NewBPFCollector(),
 		observer.NewBPFCollector(),
 		process.NewBPFCollector(),
+		bpfmetrics.NewBPFCollector(),
```

Comment:
This collector can be registered separately, like that:
```
registry.MustRegister(bpfmetrics.NewBPFCollector())
```

The map metrics collectors must be wrapped because they share metric descriptors (see [this comment](https://github.com/cilium/tetragon/blob/18d1b4ec2498c68a6d704dba0c1abca0472ccbe7/pkg/metrics/mapmetrics/mapmetrics.go#L41-L44)). But this new collector doesn't share metric descriptors with any other one.

Reply:
Ok, thanks. I will register that separately.